matador/utils/chem_utils.py

```diff
@@ -643,3 +645,36 @@ def get_formula_from_stoich(stoich, elements=None, tex=False, sort=True, latex_s
                 else:
                     form += elem[0] + str(int(elem[1]))
     return form
+
+def magres_reference_shifts(
+    magres: list,
+    reference: dict,
+    shielding_key: str = "chemical_shielding_iso",
+    shift_key: str = "chemical_shift_iso",
+):
+    """ Convert chemical shielding to chemical shift.
+
+    Parameters:
+        magres (list): list of containing magres data.
```

Comment:
```suggestion
        magres (Dict[str, Any]): A matador document containing the structure and magres shielding data.
```

Reply:
Actually, if you have the type hints set correctly, then all type annotations within the docstring can be removed